Source/NSApplication.m

```diff
@@ -542,6 +547,11 @@ - (BOOL) acceptsFirstMouse: (NSEvent*)theEvent
   return YES;
 }
 
+- (BOOL) needsPanelToBecomeKey
```

Comment:
I don't see why this change is needed. NSView already should return NO for this method so we are just duplicating code from the super class.
The other part of this change looks good to me.

Reply:
OK, let me check NSView code.

Reply:
You are right. I've removed `needsPanelToBecomeKey` override.

Reply:
By the way, what window manager do you use? I need to know to do additional checks before submitting PR to -gui and -back.

Reply:
I normally use kwin, which comes with the KDE desktop environment. Years ago I tried to test GNUstep code with a lot of different window managers but I gave up on that long ago.

Reply:
I understand you. I guess you just don't see some appicon flickering (GSSuppressAppIcon?). I have some questions on Cairo backend implementation. What is better the method for it: PM or mail-list?